lib/ansible/modules/packaging/language/maven_artifact.py

```diff
@@ -441,6 +462,10 @@ def main():
     dest = module.params["dest"]
     b_dest = to_bytes(dest, errors='surrogate_or_strict')
     keep_name = module.params["keep_name"]
+    verify_checksum = module.params["verify_checksum"]
+
+    verify_download = verify_checksum == 'download' or verify_checksum == 'always'
```

Comment:
Nothing wrong, but afaik it's a bit more pythonish and concise to write
`verify_download = verify_checksum in ['download', 'always']`